src/containers/Home/index.jsx

```diff
@@ -2,13 +2,16 @@ import { useState } from 'react';
 
 import Input from '../../components/Input';
 
```

Comment:
Please remove this empty line. Custom modules should be imported altogether in the same block.
Just to make sure we're on the page here, this is the convention we've been trying to follow:

```
imports of external libraries (e.g. react, prop-types, etc.)
empty line
imports of custom modules (e.g. own react componentes)
empty line
imports of styles
```

For instance: 
```
import { useState } from 'react';

import Input from '../../components/Input';
import Dropdown from '../../components/Dropdown';

import styles from './styles.module.css';
```

Reply:
LMK if this is not clear and we can discuss it.

Reply:
Got it